xbmc/platform/win32/threads/ThreadImplWin.cpp

```diff
@@ -60,29 +60,34 @@ CThreadImplWin::CThreadImplWin(std::thread::native_handle_type handle) : IThread
 
 void CThreadImplWin::SetThreadInfo(const std::string& name)
 {
-  const unsigned int MS_VC_EXCEPTION = 0x406d1388;
-
-#pragma pack(push,8)
-  struct THREADNAME_INFO
+  // Modern way to name threads first (Windows 10 1607 and above)
+  if (!CWIN32Util::SetThreadName(reinterpret_cast<HANDLE>(m_handle), name))
```

Comment:
```suggestion
  if (!CWIN32Util::SetThreadName(static_cast<HANDLE>(m_handle), name))
```

Same here then (it is preferable use `static_cast` whenever it is sufficient)

Reply:
yes, done